about/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Ромикс</title>
+</head>
+<body>
+    <h1>Об Авторе</h1>
+    <h2>Почему я выбрал Front-end</h2>
+    <p>Всем привет, моё имя Роман, я учусь в направление Front-end разработки</p>
+    <p>Вы спросите, почему я выбрал именно это направление выбрал?
+        <br>В моём учреждении образования есть предмет ПССИП (Программные средства создания Internet-приложений).
+        <br>Он мне очень сильно понравился. Вот поэтому я решил выбрать направление Front-end разработки.</p>
+    <blockquote>Без любви в наше время, ни-ку-да...</blockquote>
+    <h3>Какие у меня качества разработчика</h3>
+    <p><strong>Я усидчивый,</strong><em>мышление</em><ins>развито неплохо</ins><mark>(умею решать задачи),</mark><del> неплохо</del><sup> работаю</sup><sub> в команде.</sub></p>
```

Comment:
Please add spaces for this line in the next Pull request, the words are merged into one :)
Otherwise the work looks good